package.json

```diff
@@ -126,17 +132,28 @@
   },
   "jest": {
     "collectCoverage": true,
+    "clearMocks": true,
     "coverageReporters": [
       "lcov"
     ],
     "modulePathIgnorePatterns": [
-      "<rootDir>/packages/*.*/dist/*.*"
+      "<rootDir>/packages/*.*/dist/*.*",
+      "<rootDir>/packages/*.*/public/*.*",
+      "<rootDir>/packages/*.*/.cache/*.*"
     ],
```

Comment:
i think our `prettier` path regex can now be updated to:
```
"prettier": "prettier --write --single-quote --trailing-comma=none 'storybook/**/*.js' 'packages/**/src/**/*.js'"
```
etc... (please add any other files I am missing)